rest_framework/utils/serializer_helpers.py

```diff
@@ -82,6 +83,12 @@ def as_form_field(self):
         return self.__class__(self._field, value, self.errors, self._prefix)
 
 
+class JSONBoundField(BoundField):
+    def as_form_field(self):
+        value = json.dumps(self.value)
```

Comment:
That's a good point.  Right now it does input, but I can see if I can make it do indented + textarea.